src/components/text-editor/link-menu/editor-link-menu.tsx

```diff
@@ -17,7 +26,7 @@ import { LimelInputFieldCustomEvent } from '../../../components';
  */
 @Component({
     tag: 'limel-text-editor-link-menu',
-    shadow: true,
+    shadow: { delegatesFocus: true },
```

Comment:
I think this line, **or perhaps something else**, might be causing a problem:

https://github.com/Lundalogik/lime-elements/assets/35954987/8391c9a6-9ab1-4db1-a165-909bb83920fa

when you type in the second input field and pause for a second, the focus jumps to the first input field (perhaps as the component re-renders). If it's this line, the `delegateFocus` and `tabindex` are working. If it's not these, then the problem might be coming from `limel-menu-surface`.

Reply:
Are you referring to the input field being shown as invalid? I think that is on main too 😅

https://github.com/orgs/Lundalogik/projects/149/views/56?pane=issue&itemId=66149636 there's an issue for it

Reply:
I think he's referring both to that and the focus being set on the first input. I think it's due to the menu-surface being re-rendered which triggers a re-render of the input fields, and this in turn then sets the focus to the first input field.

Reply:
I've fixed the focus issue in a subsequent fixup.